README.md

```diff
@@ -161,6 +161,34 @@ will be added as they are required.
 
 ---
 
+`should_not_redirect`
```

Comment:
I'm not sure in what case you would need to do this.  Since each request can only be responded to by the F5 once, wouldn't using one of the other tests that call out what you want it to do be better?  For instance "should_select_pool"?  Am I missing a use case here?